.gitignore

```diff
@@ -0,0 +1,48 @@
+# System
+.DS_Store
+
+# IDE
+.git/
+.vscode/
+.idea/
+
+# Logs
+logs
+*.log
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
+
+# Runtime data
+pids
+*.pid
+*.seed
+*.pid.lock
+
+# Coverage directory used by tools like istanbul
+coverage
+
+# Dependency directories
+node_modules/
+
+# Optional npm cache directory
+.npm
+
+# Optional eslint cache
+.eslintcache
+
+# Output of 'npm pack'
+*.tgz
+
+# Yarn Integrity file
+.yarn-integrity
+
+# dotenv environment variables file
+.env
+
+# WebPack Cache
+.cache-loader
+
+# Project
+/config.js
+/public/
```

Comment:
i don't see a folder named "public". Is this supposed to be inhere?

Reply:
removed